removal_from_ansible.rst

```diff
@@ -0,0 +1,47 @@
+*****************************************************
+Ansible Community Package Collections Removal Process
+*****************************************************
+
+.. contents:: Topics
+
+Overview
+========
+
+This document describes under which circumstances collections can be removed from the `Ansible community package <https://pypi.org/project/ansible/>`_ (`build data <https://github.com/ansible-community/ansible-build-data/>`_). Right now this document tries to set up some first processes. In cases of emergency the `Ansible Community Engineering Steering Committee <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_ can vote on emergency exceptions, but the goal is to only use processes documented in here.
+
+Removing broken collections
+===========================
+
+If it turns out that a collection contained in Ansible X.0.0 depends on another collection included in X.0.0 and does not work with it, the collection can be removed from Ansible (X+1).0.0 under the following conditions:
+
+1. The collection seems to be unmaintained, and nobody successfully manages to fix the problems.
+2. The imminent removal in the next major Ansible release is announced in the Ansible changelog, in The Bullhorn, and in the collection's issue tracker at least two months before the (X+1).0.0 release, and at least one month before the first (X+1).0.0 beta release.
```

Comment:
```suggestion
2. The imminent removal in the next major Ansible release and its reasons are announced in the Ansible changelog, in The Bullhorn, and in the collection's issue tracker at least two months before the (X+1).0.0 release, and at least one month before the first (X+1).0.0 beta release. The announcement must also contain information that, to prevent the removal, the collection urgently needs new maintainers who can fix the problems.
```
Or something with similar meaning.